scss/_patterns_table-mobile-card.scss

```diff
@@ -26,46 +27,43 @@
         flex-direction: column;
         margin-bottom: $spv-outer--scaleable;
         overflow: auto; // prevent overflow of child margins
-        padding: 0 $sph-inner--large calc(#{$spv-inner--large} - 1px);
-        width: calc(50% - #{0.5 * map-get($grid-gutter-widths, medium)});
+        padding: 0 $sph-inner;
+        // width: calc(50% - #{0.5 * map-get($grid-gutter-widths, medium)});
```

Comment:
Remove if not needed.

Reply:
oh I hadn't saved that file, it is gone now.